README-PUBLIC.md

```diff
@@ -19,15 +19,28 @@ make lint
 
 **To set up PHPUnit locally** (requires a working WordPress development environment, specifically PHP and MySQL):
 
-Notes:
```

Comment:
Hm, I'm not sure why these were originally added to README-PUBLIC; I think they should be in the main README instead.

Reply:
Should README-PUBLIC.md just be fully combined with the main README? If we remove the testing instructions the only unique content in that file will be a sentence about what this repo is and a link to the repo itself.

Reply:
Yeah, I moved most of the content to README. The PUBLIC one is used here: https://github.com/Automattic/vip-go-mu-plugins-built